wgpu-hal/src/gles/egl.rs

```diff
@@ -572,7 +586,7 @@ impl Inner {
         } else {
             khronos_egl::OPENGL_ES_API
         })
-        .unwrap();
+        .map_err(instance_err("Failed to bind API"))?;
```

Comment:
nit: inconsistent casing; most of these errors start lower case, let's stick with that
```suggestion
        .map_err(instance_err("failed to bind API"))?;
```